examples/1_startandwait.py

```diff
@@ -0,0 +1,18 @@
+from tagilmo.utils.malmo_wrapper import MalmoConnector
+import tagilmo.utils.mission_builder as mb
+
+
+miss = mb.MissionXML()
+# just an empty flat world for testing
+miss.setWorld(mb.flatworld(""))
```

Comment:
Doesn't work for me. I'd use the default world here, but with FIXME

Reply:
Weird, that works for me. Anyway, done.